protonvpn_cli/logger.py

```diff
@@ -11,7 +11,7 @@ def get_logger():
     Always logs to file and to console when using PVPN_DEBUG=1
```

Comment:
build.build.sh runs a fmt, yapf, and SA, mypy. All fmt changes where done by yapf. Extra comments were added by me as exceptions to mypy.